### ISSUE TITLE
[BUG] runtime generated with single --platform ends up in wrong folder

### ISSUE THREAD

**Issue by EmberLightVFX** [dashingsoft/pyarmor#1511]
```
INFO     Python 3.9.13
INFO     Pyarmor 8.3.8 (group), 006099, aesl
INFO     Platform windows.x86_64
```
If I generate an obfuscation with python 3.9 together with `--platform windows.x86_64`, instead of the runtime ending up in the folder `src\pyarmor_runtime_006099\py39` it ends up in `src\py39\pyarmor_runtime_006099`. The pyarmor and py39 folder have bin flipped.

Because of this the generated script can't be launched as it doesn't find the pyarmor runtime.

If I add an extra platform so I end up with `--platform windows.x86_64 --platform darwin.x86_64` the folder structure ends up correct and the script works.

**Comment by jondy** [comment]
Could you describe issue by full command line options, expected results and actual results?

**Comment by EmberLightVFX** [comment]
`pyarmor --debug gen --output C:\\Users\\jacob\\Syncthing\\Projekt\\Python\\aesl -r -i --platform windows.x86_64` generates the runtime files here: `aesl\py39\pyarmor_runtime_006099`

while `pyarmor --debug gen --output C:\\Users\\jacob\\Syncthing\\Projekt\\Python\\aesl -r -i --platform windows.x86_64 --platform darwin.x86_64` generates the runtime files here: `aesl\pyarmor_runtime_006099\py39`

Notice that `py39` and `pyarmor_runtime_006099` have switched places. `pyarmor_runtime_006099\py39` is the correct order.

**Comment by jondy** [comment]
Got it, the problem is in the plugin `MultiplePythonPlugin`.

It will be fixed in next release.

**Comment by jondy** [comment]
Can't reproduced.

I test it by this way
```
$ pyarmor cfg plugins += "MultiPythonPlugin"
$ pyarmor gen -i tsrc/joker
$ ls dist
```

The runtime path is at `dist/joker/pyarmor_runtime_005068/py312/`

The MultiPythonPlugin is in the open source `/path/to/pyarmor/cli/plugins.py`, you can also add some print statement in this plugin to find the problem.

**Comment by utjduo** [comment]
If you don't enter any --platform you won't get the bug. But I might need to use --platform windows.x86_64 on a from a mac computer and that will end up with the bug

**Comment by EmberLightVFX** [comment]
If you don't enter any --platform you won't get the bug. But I might need to use --platform windows.x86_64 on a from a mac computer and that will end up with the bug

**Comment by jondy** [comment]
Reproduced, it will be fixed in next release.

**Comment by jondy** [comment]
Here it's workaround patch 
6ed4e11902e169ae677ffdc6e6c804bb8d2313c7

**Comment by jondy** [comment]
Fixed in 8.3.10
